product_docs/docs/pgd/5/cli/installing_cli.mdx

```diff
@@ -8,23 +8,23 @@ TPAexec installs and configures PGD CLI on each BDR node, by default. If you wis
 
 ## Installing manually
 
-You can manually install the PGD CLI on any Linux machine using `.deb` and `.rpm` packages available from the [BDR repository](https://techsupport.enterprisedb.com/software_subscriptions/add/products/bdr4/). The package name is `edb-pgd-cli`. For example:
+You can manually install the PGD CLI on any Linux machine using `.deb` and `.rpm` packages available from the EDB repository. The package name is `edb-pgd5-cli`. For example:
 
 ```sh
 # for Debian
-sudo apt-get install edb-pgd-cli
+sudo apt-get install edb-pgd5-cli
 ```
 
 When the PGD CLI is configured by TPAexec, it connects automatically, but with a manual installation to a standalone EDB Postgres Distributed cluster you need to provide a connection string.
 
 
 ### Specifying database connection strings
 
-You can either use a configuration file to specify the database connection strings for your cluster (see following section) or pass the connection string directly to a command (see the [sample use case](/pgd/latest/cli/#passing-a-database-connection-string)). 
+You can either use a configuration file to specify the database connection strings for your cluster (see following section) or pass the connection string directly to a command (see the [sample use case](/pgd/latest/cli/#passing-a-database-connection-string)).
```

Comment:
remove (see following section) OR make it a link to that section.
Would be better as two sentences:
You can use a configuration file to specify the database connection strings for your cluster. Alternatively, you can pass the connection string directly to a command. For details, see the [sample use case](/pgd/latest/cli/#passing-a-database-connection-string).

Reply:
yes, thanks. fixed.